private/shell/utils/index.ts

```diff
@@ -237,12 +244,9 @@ export const prepareStyles = async (
     ? await inputFile.text()
     : `@import 'tailwindcss';`;
 
-  const basePath = isPackageLinked()
```

Comment:
I will look into another separate change for this, since it breaks cases where a `styles.css` file is present.